compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/analyzer/DataflowAnalyzer.java

```diff
@@ -1656,15 +1667,21 @@ public void visit(BLangErrorVarRef varRefExpr) {
     @Override
     public void visit(BLangTupleVariable bLangTupleVariable) {
         analyzeNode(bLangTupleVariable.typeNode, env);
+        if (bLangTupleVariable.expr != null) {
+            this.currDependentSymbol.push(bLangTupleVariable.symbol);
+            analyzeNode(bLangTupleVariable.expr, env);
+            this.currDependentSymbol.pop();
+            return;
+        }
+        bLangTupleVariable.memberVariables.forEach(member -> {
+            analyzeNode(member, env);
+        });
     }
 
     @Override
     public void visit(BLangTupleVariableDef bLangTupleVariableDef) {
         BLangVariable var = bLangTupleVariableDef.var;
-        if (var.expr == null) {
-            addUninitializedVar(var);
-            return;
-        }
+        analyzeNode(var, env);
```

Comment:
We no longer need the `var` variable, right?

```suggestion
        analyzeNode(bLangTupleVariableDef.var, env);
```